tests/operators/test_file_operators.py

```diff
@@ -0,0 +1,54 @@
+#!python3
+# test_http_to_file_operators - test suite for http to file operators
```

Comment:
Same comment as the other test file. Do we need the shebang? And the name of this test suite is different than the file. Mistake?